_drafts/SC2-1-2-no-keyboard-trap-r1.md

```diff
@@ -0,0 +1,38 @@
+---
+name: No keyboard trap R1
+group:
+- SC2-1-2-no-keyboard-trap-r1 (current)
+- SC2-1-2-no-keyboard-trap-r2
+description: This rule checks if it is possible to navigate through all content on a web page using a keyboard without becoming trapped in any element or group of elements.
+success_criterion: 
+- 2.1.2
+test aspects: DOM Tree
+authors:
+- Dagfinn Rømen
+- Geir Sindre Fossøy
+- Malin Øvrebø
+- Shadi Abou-Zahra
+- Carlos Duarte
+- Anne Thyme Nørregaard
+---
+
+## Test procedure
+### Applicability
+The rule applies to all elements on a web page where focus can be moved to through keyboard navigation.
+
+### Expectation
+For each target element, focus can be moved to the next element on the web page by using arrow keys, ESC key, TAB key, Shift+TAB or Enter key.
```

Comment:
I don't think the logic of this works. Firstly because the "next element on the page" can be any element, I was going to suggest the "next element in the focus order", but I don't think that's true either. In things like ARIA tabs the next tab shouldn't be in the focus order. You should make sure this rule doesn't become a test that checks that the focus order makes sense. That's entirely different SC, and not the problem you are trying to describe.

This one is difficult to review without knowing what is going to be in the R2 rule.